### ISSUE TITLE
Support for non standard SSH port

### ISSUE THREAD

**Issue by xezpeleta** [elkarbackup/elkarbackup#87]
Currently Elkarbackup is using SSH default port (tcp/22) and there is no way to specify a custom port. Apparently in Rsnapshot you can do that using the following syntax:

```
backup /var/ localhost/ ssh_args=-p456
```

So maybe we can include this feature in Elkarbackup.

**Comment by xezpeleta** [comment]
A better way could be to add a field "ssh_args" in the Client edition form, but it requires to modify the DB schema.

**Comment by goniz** [comment]
It's also possible to do that with:
"--rsh=ssh -p PORT" 
But ssh_args seems like the right option

**Comment by xezpeleta** [comment]
Done. By the moment it will be a hidden feature, so we can specify custom args in the same way we do it in Rsnapshot using the word `ssh_args`.

Client Url, listening in SSH port 2222:
```
root@192.168.33.10 ssh_args=-p2222
```

It should be valid for any ssh argument.

![captura de pantalla 2015-04-20 a las 16 54 41](https://cloud.githubusercontent.com/assets/1846038/7232819/fdc3a7e6-e77d-11e4-8f90-563ae5d26a53.png)

**Comment by xezpeleta** [comment]
To do: Test script execution

**Comment by goniz** [comment]
Wow,  great response  time :)
Hope to test it today to will report soon

**Comment by parreitu** [comment]
Hi @Xabi

I've tried to test your solution, because I think that it could be a good solution in another case, but it doesn't work for me.
![elkarbackup](https://cloud.githubusercontent.com/assets/3928521/7285704/dcd57b2a-e946-11e4-9c63-c71b96adf114.png)

Elkarbackup was reporting errors, so I've got the rsnapshot config file (http://elkarbackup/client/1/job/1/config) and this is the rsync command generated by rsnapshot in the last line of the configuration file
```
backup    root@192.168.1.137 ssh_args=-p2222:/sdcard/DCIM    .
```
and this is the error

```
root@elkarbackup:~/kk# rsnapshot -c config.txt sync
Unexpected remote arg: ssh_args=-p2222:/sdcard/Download
rsync error: syntax or usage error (code 1) at main.c(1232) [sender=3.0.9]
----------------------------------------------------------------------------
rsnapshot encountered an error! The program was invoked with these options:
/usr/bin/rsnapshot -c config.txt sync
----------------------------------------------------------------------------
ERROR: /usr/bin/rsync returned 1 while processing root@192.168.1.137 ssh_args=-p2222:/sdcard/Download
```
Of course, If I try to connect by ssh from the Elkarbackup console, it works.

So, I've been changing things, and finally I've found the error. Changing the rsync command in order to put the ssh_args in the final part of the command, all is ok.
```
backup  root@192.168.1.137:/sdcard/Download     .       ssh_args=-p2222
```
Now, the question is how to change the code in order to change the position of the ssh_args part in the rsync command

**Comment by goniz** [comment]
It works great for me.
Did you compile+install the latest code from master?

**Comment by xezpeleta** [comment]
That's it @parreitu , this feature will be available in the next version (1.1.4), so download github code or wait some weeks ;)

**Comment by xezpeleta** [comment]
Now yes, you can test the pre-release:

https://github.com/elkarbackup/elkarbackup/issues/89
